src/firefly/js/ui/CatalogSearchMethodType.jsx

```diff
@@ -47,16 +52,33 @@ export class CatalogSearchMethodType extends Component {
         });
     }
 
+    componentWillReceiveProps(nextProps) {
+        // const fields= FieldGroupUtils.getGroupFields(this.nextProps.groupKey);
+        const {coneMax, boxMax, groupKey}= nextProps;
+        if (coneMax && groupKey) {
```

Comment:
found another problem to, I accidentally took out the passing the properties when I was cleaning up.